pkg/db/db.go

```diff
@@ -44,9 +44,9 @@ func Init(db *Database) {
 	case "etcd":
 		C = etcd.NewClient(strings.Split(db.Endpoint, ","))
 		return
-	case "fake":
-		C = fakedb.NewFakeDbClient()
-		return
+	//case "fake":
```

Comment:
You can't remove these lines, because it will be used in integration test

Reply:
Thanks for review. 
You are right, we can't remove these lines.
Actually still some modifications are going on.  I will uncomment it. To just test without fake client, i had added.

Reply:
please do not remove fake db, otherwise CI will fail.

Reply:
done!